src/guides/v2.2/ui_comp_guide/components/ui-linkcolumn.md

```diff
@@ -21,3 +21,70 @@ Constructor: [app/code/Magento/Ui/view/base/web/js/grid/columns/link.js]({{ site
 Extends [Column component]({{ page.baseurl }}/ui_comp_guide/components/ui-column.html)
 
 -  [`app/code/Magento/Ui/view/base/web/js/grid/columns/link.js`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/js/grid/columns/link.js)
+-  [`app/code/Magento/Ui/view/base/web/templates/grid/cells/link.html`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/templates/grid/cells/link.html)
+
+## Examples
+
+### Integrate the LinkColumn component with the Listing component
+
+This example integrates the LinkColumn component with the [Listing]({{ page.baseurl }}/ui_comp_guide/components/ui-listing-grid.html) component:
+
+```xml
+<listing>
+    <columns>
+        <column name="id" component="Magento_Ui/js/grid/columns/link" sortOrder="10">
+            <argument name="data" xsi:type="array">
+                <item name="config" xsi:type="array">
+                    <item name="link" xsi:type="string">link</item>
+                </item>
+            </argument>
+            <settings>
+                <label translate="true">Link</label>
+                <sortable>false</sortable>
+            </settings>
+        </column>
+    </columns>
+</listing>
+```
+
+Data Provider for this listing:
+
+```php
+<?php
+
+namespace Vendor\Module\Ui\Component\Listing;
+
+use Magento\Framework\View\Element\UiComponent\DataProvider\DataProvider;
+
+/**
+ * Class Actions
+ */
+class CustomDataProvider extends DataProvider
+{
+    /**
+     * Get data
+     *
+     * @return array
+     */
+    public function getData()
+    {
+        return [
+            'items' => [
+                [
+                    'id' => 1,
+                    'link' => 'https://magestore.com/path/to/controller/id/1'
+                ],
+                [
+                    'id' => 2,
+                    'link' => 'https://magestore.com/path/to/controller/id/2'
```

Comment:
I would kindly ask you to use another URL for the example. "Magestore" is the name of the existing extension vendor. Thank you!